easybuild/easyconfigs/p/PyQt/PyQt-4.11.3-goolf-1.5.14-Python-2.7.6.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'PyQt'
+version = '4.11.3'
+versionsuffix='-Python-2.7.6'
+
+homepage = 'http://www.riverbankcomputing.co.uk/software/pyqt'
+description = """
+PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework 
+and runs on all platforms supported by Qt including Windows, MacOS/X and Linux. 
+PyQt5 supports Qt v5. PyQt4 supports Qt v4 and will build against Qt v5. 
+The bindings are implemented as a set of Python modules and contain over 620 classes. 
+"""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+sources = ['%(name)s-x11-gpl-%(version)s.tar.gz']
+source_urls = ['http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3']
+
+python = 'Python'
+pythonver = '2.7.6'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+dependencies = [
+    (python, pythonver),
+    ('SIP','4.16.4',versionsuffix),
+    ('Qt','4.8.4')
+]
+
+
+configopts = " configure-ng.py --confirm-license "
+    += "--destdir=%%(installdir)s/lib/python%s/site-packages " %pythonshortver
+    += "--no-sip-files"
```

Comment:
you need to use `configopts +=` to get this to work, right?

also: space after `%` in `%pythonshortver`